playbooks/byo/openshift-cluster/redeploy-certificates.yml

```diff
@@ -14,7 +14,36 @@
   gather_facts: no
   tasks:
   - include_vars: ../../byo/openshift-cluster/cluster_hosts.yml
+  - set_fact:
+      openshift_deployment_type: "{{ deployment_type }}"
 
-- include: ../../common/openshift-cluster/redeploy-certificates.yml
-  vars:
-    openshift_deployment_type: "{{ deployment_type }}"
+- include: ../../common/openshift-cluster/evaluate_groups.yml
+  tags:
+  - always
+
+- include: ../../common/openshift-cluster/initialize_facts.yml
+  tags:
+  - always
+
+- include: ../../common/openshift-cluster/initialize_openshift_version.yml
+  tags:
+  - always
```

Comment:
It looks like everything from line 2 to here is boilerplate that is common to all of the playbooks, if it's being included in each of the included playbooks, it shouldn't be needed here. I'm also wondering if it would make sense to have a playbook that encapsulates these common steps to avoid the boilerplate feel.